plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -291,11 +291,11 @@ public boolean isPartitionStatsIndexEnabled()
         return isPartitionStatsIndexEnabled;
     }
 
-    @Config("hudi.dynamic-filtering-wait-timeout")
-    @ConfigDescription("Maximum timeout to wait for dynamic filtering to collect complete its collection in SECONDS")
-    public HudiConfig setDynamicFilteringWaitTimeout(long dynamicFilteringWaitTimeout)
+    @Config("hudi.dynamic-filtering.wait-timeout")
+    @ConfigDescription("Maximum timeout to wait for dynamic filtering")
```

Comment:
Let's add docs on what's expected as the config value, e.g., "1s" in the duration format